core/src/main/java/io/grpc/internal/TransportSet.java

```diff
@@ -346,8 +333,18 @@ public void transportShutdown(Status s) {
           new Object[] {transport, address});
```

Comment:
I think it's okay for now to mostly lose the statuses for earlier failed transports. However, it would probably be a good idea to add `s` to this log output in case someone needs to debug why earlier addresses failed.